action.yml

```diff
@@ -18,6 +18,11 @@ inputs:
       One of: MERGE, STASH or REBASE'
     required: false
     default: SQUASH
+  PRESET:
+    description:
+      'Enable additional functionality to better personalize the behavior.`
+      DEPENDABOT_MINOR or DEPENDABOT_PATCH'
```

Comment:
I think we forgot to reformat this description when changing it?

Reply:
Fixed!